src/world_diff.rs

```diff
@@ -26,11 +27,13 @@ pub struct WorldDiff {
 
     // The fields below are only rolled back when the whole VM is rolled back.
     pub(crate) decommitted_hashes: RollbackableSet<U256>,
-    read_storage_slots: RollbackableSet<(H160, U256)>,
-    written_storage_slots: RollbackableSet<(H160, U256)>,
+    pub read_storage_slots: RollbackableSet<(H160, U256)>,
+    pub written_storage_slots: RollbackableSet<(H160, U256)>,
```

Comment:
Dumb question: Why are these fields public? AFAIU, they are not accessed in the Era codebase, only `storage_application_cycles` is.

Reply:
Noticed the same while rebasing. Should be fixed now.